markdown_xblock/html.py

```diff
@@ -56,6 +64,7 @@ class MarkdownXBlock(StudioEditableXBlockMixin, XBlockWithSettingsMixin, XBlock)
         default=_('Markdown')
     )
     classes = List(
+        display_name=_('Classes'),
         help=_('Custom CSS classes to add to this component'),
```

Comment:
I think you wanted to add some information in this help text to indicate that this must be a JSON list.

Maybe something like:

```python
         help=_('JSON list of strings representing custom CSS classes to add to this component'),
```

or

```python
         help=_('Custom CSS classes to add to this component (must be a valid JSON list)'),
```

Reply:
Yes, I did now, thanks! :)